Gravatar/MruImageCache.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+
+namespace Gravatar
+{
+    /// <summary>
+    /// Caches most-recently-used images.
+    /// </summary>
+    /// <remarks>
+    /// Decorates an inner cache, delegating to it as needed.
+    /// <para />
```

Comment:
:memo: The use of `para` is inconsistent again. I have a set of analyzers to help with this that could be made public if it would help.

Reply:
If this is something we care about then let's add an analyzer in a different PR so that we don't spend time covering it in reviews.